### PR TITLE
fix: display curl in the webhook page

### PR DESCRIPTION
**Pull request by likyh**
# Summary

display curl in the webhook page

kBKVYCqVbG

The text copied is:
```
curl http://localhost:4000/api/plugins/webhook/1/deployments -X 'POST' -d "{
  \"repo_url\":\"$CIRCLE_REPOSITORY_URL\",
  \"commit_sha\":\"$CIRCLE_SHA1\",
  \"start_time\":\"$start_time\"
}"
```

### CODE REVIEW

**Comment by mintsweet** [comment]
@likyh The style of the url display is broken.

**Comment by likyh** [comment]
bDH3P7rcDM

**Comment by mintsweet** [comment]
LGTM.

**Comment by likyh** [comment]
image
fix some wording again.

File: config-ui/src/pages/connections/incoming-webhook/styled.js [hunk 1]
```diff
@@ -143,10 +143,14 @@ export const FormWrapper = styled.div`
       display: flex;
       align-items: center;
       justify-content: space-between;
-      padding: 0 10px;
-      height: 38px;
+      padding: 10px;
+      height: auto;
```
Comment (mintsweet) [inline config-ui/src/pages/connections/incoming-webhook/styled.js:147]: This property can not be set.

Comment (likyh) [reply config-ui/src/pages/connections/incoming-webhook/styled.js]: fixed

**Comment by mintsweet** [comment]
LGTM.

**Review by Startrekzky — APPROVED** [review]
LGTM

**Pull request merged**